guides/SPI.md

```diff
@@ -18,9 +18,22 @@ Once installed and enabled on a site, this provider will be called to fetch appl
 
 ## Setup
 In order to become a _service provider_, you have two options:
-* Register an application in the Wix Developer Center, and add the SPI component that you wish to implement. 
+* Register an application in the Wix Developers Center, and add the integration component that you wish to implement. 
```

Comment:
add something that ties this in to the instructions, e.g., "(instructions below)"